msteams-platform/toolkit/configure-message-extension-capability.md

```diff
@@ -0,0 +1,634 @@
+---
+title: Configure message extension capability
+author: surbhigupta
+description: Learn how to configure message extension capability within your Teams app.
+ms.author: v-vanv
+ms.localizationpriority: medium
+ms.topic: overview
+ms.date: 03/01/2024
+---
+
+# Configure message extension capability within your Teams app
+
+Message extension allows users to interact with your web service when composing messages in Microsoft Teams. Users can invoke your web service to assist message composition from the message compose box or from the search bar.
+
+Message extensions are implemented on top of the bot support architecture within Teams. For more information, see [Build message extensions](../messaging-extensions/what-are-messaging-extensions.md).
+
+## Prerequisites
+
+To configure message extension as an additional capability, ensure the following prerequisites are met:
+
+* The app manifest (previously called Teams app manifest) file.
+* [Microsoft 365 account](../concepts/build-and-test/prepare-your-o365-tenant.md) to test the application.
+* [Microsoft Azure Storage account](/azure/storage/common/storage-account-create).
+
+## Add message extension to Teams app
+
+# [Add message extension to Teams tab app](#tab/tabapp)
+
+The following steps help you to add message extension to a tab app:
+
+1. [Create message extension app using Microsoft Teams Toolkit](#create-message-extension-app-using-microsoft-teams-toolkit)
```

Comment:
When we are adding ME to the app how can we Create message extension app?

It should be Create tab app using Microsoft Teams Toolkit